e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js

```diff
@@ -185,6 +187,38 @@ describe(
         });
       },
     );
+
+    it("should show detailed form errors for constraint violations when executing model actions", () => {
+      const actionName = "Update";
+
+      cy.signInAsAdmin();
+
+      cy.get("@modelId").then(modelId => {
+        cy.visit(`/model/${modelId}/detail`);
+        cy.wait("@getModel");
```

Comment:
```suggestion
```

I don't see the point of going to the model detail page. `createImplicitActions` creates basic actions (editing) via API, so there's no need to visit any page before that. This would eliminate the need to intercept `getModel` too.

Reply:
41176fc9fcbd8c0dd9698e33eedc165002b58b28